client/webserver/site/src/css/forms.scss

```diff
@@ -1,5 +1,5 @@
 #regAssetForm {
-  position: relative;
+  max-width: 425px;
```

Comment:
This is causing new wrapping in the lot size column of the markets list, the net effect being only one market row is shown without scrolling instead of two.

before:
![image](https://github.com/decred/dcrdex/assets/6109680/11df312f-9605-4720-a9dc-221aaab6a7e0)

after:
![image](https://github.com/decred/dcrdex/assets/6109680/eb164556-5efd-4667-8425-d90935a9d59e)


Looks like it wants to grow to about 430px. I do think that 425 should be a reasonable max width for form elements, so we probably need some layout work here. For now, can you just assign the max width to the new whatsabond element? Putting `text-nowrap` on some elements seems to work too.